### DISCUSSION TITLE
Transient Analysis

### DISCUSSION THREAD

**Discussion by rdzjr1997** [petrobras/ross#946] [Q&A]
Is there any option for transient analysis at ROSS? 
By transient analysis I mean rotation varying with time (non-stationary analysis), as a run-up or a run-down of the rotor

**Comment by raphaeltimbo** [comment]
> Is there any option for transient analysis at ROSS? By transient analysis I mean rotation varying with time (non-stationary analysis), as a run-up or a run-down of the rotor

We do not have this capability yet. To do that, we would need to add some additional arguments to our methods, such as the rotor acceleration rate and we would also need to pass this to methods that assemble the elements matrices. Currently, we are evaluating the implementation of this feature.
